tests/lib/rules/indent.js

```diff
@@ -1208,6 +1288,8 @@ ruleTester.run("indent", rule, {
                 [120, 4, 6, "VariableDeclaration"],
                 [124, 4, 2, "BreakStatement"],
                 [134, 4, 6, "BreakStatement"],
+                [138, 2, 3, "Punctuator"],
```

Comment:
I might be misunderstanding your suggestion here. The way I see it, if you have this code:

```javascript
/*eslint indent: ["error", 2, { "MemberExpression": 1 }]*/

foo
 .bar;
```

...then if we specify the Punctuator, we get an easy-to-understand and accurate error message:

>    4:2  error  Expected indentation of 2 space characters but found 1                 indent

But if we report on `MemberExpression` instead of `Punctuator` in this scenario, won't it instead say something like this?:

>   4:3  error  Expected indentation of 2 space characters but found 2                 indent